ui/text_displayer.js

```diff
@@ -268,11 +268,15 @@ shaka.ui.TextDisplayer = class {
       captionsStyle.margin = '0';
     }
 
+    const {convertLengthValue_} = shaka.ui.TextDisplayer;
```

Comment:
You only call this once.  Just refer to it directly with its full name at the call site.

Reply:
done